app/src/main/java/com/androidapp/taskmaster/activities/TaskDetailsActivity.java

```diff
@@ -25,7 +25,7 @@ protected void onResume () {
         super.onResume();
```

Comment:
I don't believe you need an onResume() here. onResume() holds an activity's state so you can navigate away and then come back (like the MainActivity remembering your name and task list). However, this taskDetailsActivity is designed to re-render itself every time you click a new task, so you don't need to preserve its state.